src/types/Renderable.ts

```diff
@@ -0,0 +1,87 @@
+// There is a difference between the core enonic types and what Guillotine returns:
+import type {
+	ComponentDescriptor,
+	// LayoutComponent,
+	LiteralUnion,
+	// PageComponent,
+	RequestMode,
+	TextComponent,
+} from '@enonic-types/core';
+import type { TextBaseProps } from './TextBaseProps';
+
+export type XpRunMode = 'development' | 'production';
```

Comment:
Should probably be moved to constants?

Reply:
It's not a value, it's just a type, which should probably be in @enonic-types/core